CMakeLists.txt

```diff
@@ -4,8 +4,12 @@ cmake_minimum_required(VERSION 2.8.12)
 # Set the project name
 project(CoreCLR)
 
-# Enable @rpath support for shared libraries.
-set(MACOSX_RPATH ON)
+set(CORECLR_SET_RPATH:BOOL ON)
```

Comment:
I have not found an option to specify type in the cmake doc. I can only see that you can specify type on cached variables, but the syntax is different.